saleor/order/calculations.py

```diff
@@ -141,11 +141,17 @@ def _recalculate_prices(
 
     order.tax_error = None
 
-    # propagate the order level discount on the prices without taxes.
+    # Flat rates, plugin and tax exemption strategies require to propagate order-level
+    # discounts to order lines. Tax app does it itself.
+    is_tax_app_strategy = (
```

Comment:
I wonder if this should be called differently like just e.g. `using_tax_app`. Adding `_strategy` may be misleading, as tax_app_strategy includes usage of plugin.